app/src/main/java/com/example/playlistmaker/presentation/media/MediaViewPageAdapter.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.playlistmaker.presentation.media
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.lifecycle.Lifecycle
+import androidx.viewpager2.adapter.FragmentStateAdapter
+
+class MediaViewPageAdapter(fragmentManager: FragmentManager, lifecycle: Lifecycle) :
+    FragmentStateAdapter(fragmentManager, lifecycle) {
+
+    override fun getItemCount() = FRAGMENTS_NUMBER
+
+    override fun createFragment(position: Int): Fragment {
+        return when(position) {
+            0 -> MediaFragmentFavourite.newInstance()
+            1 -> MediaFragmentPlaylist.newInstance()
+            else -> Fragment()
```

Comment:
```suggestion
            else -> MediaFragmentPlaylist.newInstance()
```
Здесь лучше так написать, чтобы не было создания пустого фрагмента, т.к. такой ситуации не должно происходить, поэтому лучше из кода тоже это удалить